### PR TITLE
basic about page created

### PR DESCRIPTION
**Pull request by piranha73**
# about page

added about page, with basic information

image

### CODE REVIEW

File: app/views/pages/about.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h2>I am the about page</h2>
```
Comment (ggupta78) [inline app/views/pages/about.html.erb]: Please change this to an h1

Comment (ggupta78) [inline app/views/pages/about.html.erb]:
```suggestion
<h1>I am the about page</h1>
```

**Review by ggupta78 — COMMENTED** [review]
Please take a look at a few comments added.

File: app/views/pages/about.html.erb (hunk 1 above)
Comment (piranha73) [reply app/views/pages/about.html.erb]: Thanks for spotting it

**Review by piranha73 — COMMENTED** [review]
I made the changes you suggested

**Review by ggupta78 — APPROVED** [review]

**Pull request merged**